admin/threads-seq

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env -S cargo +nightly --quiet -Zscript
+
+//! # `admin/thread-seq N`
+//!
+//! This program prints a sequence of N integers for multithreaded
+//! performance testing.  The integers are numbers of threads to
+//! be sampled in a test.  The goal is to assist in graphing
+//! how per-thread throughput relates to concurrency.
+//!
+//! The sequence is (at most) length N, starts at 2, includes the
+//! number of CPU cores, and ends at 1.5 the number of CPU cores.
+//! It does not have repeated items.
+//!
+//! We exceed the number of cores specifically to see the
+//! "elbow" in the graph, when the number of threads tested
+//! exceeds the number of cores.  (This is good because otherwise
+//! -- assuming the software under test is perfectly scalable --
+//! the graph would be a straight line parallel with the x axis.)
+
+use std::{cmp, env, thread};
+
+fn main() {
```

Comment:
I think you can use `Result<(), Box<dyn StdError + Send + Sync + 'static>>` here and use `?` more? If you're going to make it script-like I'd probably ditch the "elaborate" `expect()` and `unwrap()` calls in favor of that.